src/staff-handbook/leave/holiday.md

```diff
@@ -14,19 +13,20 @@ The holiday year at dxw runs between 1st January and 31st December, and you are
 
    This gives them a chance to manage any impact it might have on the wider team
    and the project.
-
 2. Request the holiday through BreatheHR
 
    We use BreatheHR to track who's off and when, so we can plan for it. Your
    line manager can then see and approve your request.
-
-3. Add your holiday to your Google calendar
+3. Add your holiday to your Google calendar as 'Out of office'
 
    If someone is trying to find out where you are, or when you're available, the
    first thing they'll check is your calendar.
 
    When you are on a client project, you may also need to add your holiday to a
    shared or team calendar. Check with your delivery lead.
+4. Set your Slack status to 'On holiday'
+
+   Do this when you finish on your last day before your holiday, and set it to clear when you come back.
```

Comment:
I think I've found that it sometimes gets cleared if you have Reclaim updating your status automatically, unless you manually decline all your regular meetings, but that that level of detail isn't important here